Tasks/JenkinsQueueJobV2/jobqueue.ts

```diff
@@ -187,24 +187,24 @@ export class JobQueue {
         };
 
         function walkHierarchy(job: Job, indent: string, padding: number): string {
-            let jobContents: string = indent + '<ul style="padding-left:' + padding + '">\n';
+            let jobContents: string = indent;
 
             // if this job was joined to another follow that one instead
             job = findWorkingJob(job);
 
             if (job.ExecutableNumber === -1) {
-                jobContents += indent + job.Name + ' ' + colorize(job.GetResultString()) + '<br />\n';
+                jobContents += indent + job.Name + ' ' + colorize(job.GetResultString()) + '<br />';
             } else {
                 const url = job.ExecutableUrl && job.ExecutableUrl.replace('"', '%22');
-                jobContents += indent + '<a href="' + url + '">' + job.Name + ' #' + job.ExecutableNumber + '</a> ' + colorize(job.GetResultString()) + '<br />\n';
+                jobContents += indent + '[' + job.Name + ' #' + job.ExecutableNumber + '](' + url + ') ' + colorize(job.GetResultString()) + '<br />';
```

Comment:
This line has Markdown and a `<br />` tag, which doesn't seem right to me.  Is this method supposed to return a Markdown string or an HTML string?

Reply:
The Jenkins results markdown file is a mix of markdown and html. That is what was causing the links to not show up properly.  When the link is embedded in html, it is not rendered correctly in the UI, however we need the html to create the colored success/failure badge after the link.

Reply:
The `<br />` should be fine -- all HTML is also valid Markdown.

Reply:
Well...that depends on the markdown parser that is being used.  The one we use for the build summary page doesn't like mixes of markdown and html, or at least not markdown embedded in html. :)